scripts/clean_validate.py

```diff
@@ -0,0 +1,44 @@
+# clean_validate.py
+# author: Weilin Han
+# date: 2024-10-20
+
+import click
+import os
+import sys
+import pandas as pd
+import pandera as pa
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.clean_data import extract_column_name, read_raw_data, clean_data, write_data
+from src.validate_data import build_schema_from_csv, validate_data
+
+@click.command()
+@click.option('--raw-data-file', type=str, help="Path to raw data file")
+@click.option('--name-file', type=str, help="Path to names file")
+@click.option('--data-config-file', type=str, help="Path to data configuration file")
+@click.option('--write-to', type=str, help="Path to directory where cleaned data will be written to")
+@click.option('--written-file-name', type=str, help="The name of the file will be written")
```

Comment:
`written_file_name` is a bit awkward. How about `file-name`? Or we can just hard-code this part of the code in the script. That path is what matters most about being flexible. I think I would go with the latter option.